docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc

```diff
@@ -266,11 +266,11 @@ After clicking the `Login` button, you are redirected back to the application, a
 
 The session for this demo is valid for a short period of time and, on every page refresh, you will be asked to re-authenticate.
 For information about how to increase the session timeouts, see the Keycloak https://www.keycloak.org/docs/latest/server_admin/#_timeouts[session timeout] documentation.
-For example, you can access the Keycloak Admin console directly from the dev UI by clicking the `Keycloak Admin` link if you use xref:security-oidc-code-flow-authentication.adoc#integration-testing-keycloak-devservices[Dev Services for Keycloak] in dev mode:
+For example, you can access the Keycloak Admin console directly from the dev UI by clicking the `Keycloak Admin` link if you use xref:security-oidc-code-flow-authentication.adoc#integration-testing-keycloak-devservices-b[Dev Services for Keycloak] in dev mode:
```

Comment:
This cross reference in `security-oidc-code-flow-authentication-tutorial.adoc` points to the section in `security-oidc-code-flow-authentication.adoc` whose ID is `integration-testing-keycloak-devservices-b`.

Search results for "oidc-request-filters":

```
/asciidoc/security-oidc-bearer-token-authentication.adoc:
  768:  [[integration-testing-keycloak-devservices-a]]
  870:  Use <<integration-testing-keycloak-devservices-a,Dev Services for Keycloak>> instead of `KeycloakTestResourceLifecycleManager`


/asciidoc/**security-oidc-code-flow-authentication-tutorial.adoc**:
  269:  **security-oidc-code-flow-authentication.adoc#integration-testing-keycloak-devservices-b**[Dev Services for Keycloak] in
  273:  **security-oidc-code-flow-authentication.adoc#integration-testing-keycloak-devservices-b**[Dev Services for Keycloak] section.


/asciidoc/security-oidc-code-flow-authentication.adoc:
  1801:  **[[integration-testing-keycloak-devservices-b]]**
  1843:  are encouraged to do it with <<integration-testing-keycloak-devservices-b,Dev Services for Keycloak>>.
```

Reply:
I think just the fact we are having this conversation suggest we shouldn't do it. Why is not id something sensible, like `code-flow-integration-testing-keycloak-devservices` and `bearer-integration-testing-keycloak-devservices`?

Reply:
>Why is not id something sensible, like code-flow-integration-testing-keycloak-devservices and bearer-integration-testing-keycloak-devservices?

IMHO it will avoid the confusion

Reply:
YES. If that works for you, let's do this instead!

Reply:
I’ve applied @michalvavrik's suggestion to the current PR. Please take a look and let me know if I can make any further tweaks to improve clarity.

Reply:
ids lgtm, thx